var/spack/repos/builtin/packages/py-gpy/package.py

```diff
@@ -22,4 +23,11 @@ class PyGpy(PythonPackage):
     depends_on("py-six", type=("build", "run"))
     depends_on("py-paramz@0.9.0:", type=("build", "run"))
     depends_on("py-cython@0.29:", type="build")
```

Comment:
This should technically be build/run because it's in install_requires

Reply:
I think that's incorrect on their side?

Reply:
I'm actually not sure, I see a lot of cython stuff not just in setup.py but all over the codebase. I don't know enough about cython to know how it works. You could open an issue with the devs to find out.

Reply:
They have pure python implementations, all cython stuff you see in their code is 

```python
try:
    import cython_x
    use_x_cython=True
else:
    use_x_cython=False
```

so build dep should be fine.

Reply:
Can you at least add a comment saying it is documented as build/run but is actually only needed during build? I would try to open an issue with the devs too if you have time.

Reply:
So, how should GPy list it instead?

Reply:
It should be listed under `setup_requires` instead of `install_requires`. You could even open a PR, it's a simple 2 line change